tests/unit/topology/test_graph.py

```diff
@@ -43,15 +43,7 @@ def visualize_graphs():
     ui.load_default_particles()
     topology = create_dummy_topology()
     graphs = test_initialize_graph(topology, io.load_pdg())
-    try:
-        # pylint: disable=import-error,import-outside-toplevel
-        import graphviz  # type: ignore
-
-        dot_source = io.dot.convert_to_dot(graphs)
-        vis = graphviz.Source(dot_source)
-        vis.view()
-    except ModuleNotFoundError:
-        pass
+    io.write(graphs, "jpsi_to_gamma_pi0_pi0.gv")
```

Comment:
Can any graphviz dependencies be removed now?

Reply:
You mean this one?
https://github.com/ComPWA/expertsystem/blob/2bcbb612c78bb633325ae9e7fd97a1cecbecc1a5/setup.cfg#L65

No that one is needed for the documentation.

Reply:
Just a general question, in case something can be removed

Reply:
Btw, this `try`-`except` construct was there just to facilitate the conversion of the DOT language into a PDF file, but it was actually a bit annoying to have that PDF pop up each time, so instead I now leave that conversion up to the developer.

Note that we facilitate rendering of a DOT file since #235 through a VSCode extension:
https://github.com/ComPWA/expertsystem/blob/63fa906de5a92381ba38c2d5055d910eaa324961/.vscode/extensions.json#L10